src/Message.chpl

```diff
@@ -35,13 +36,274 @@ module Message {
         var cmd: string;
         var format: string;
         var args: string;
+        var size: string; // currently unused, but wired for once all functionality moved to json
```

Comment:
again, maybe an int for size?

Reply:
Did something wrong the first time around. Updated to int and everything is working. Update following shortly.